openupgrade_scripts/scripts/crm_iap_enrich/16.0.1.1/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,6 @@
+---Models in module 'crm_iap_enrich'---
+---Fields in module 'crm_iap_enrich'---
+---XML records in module 'crm_iap_enrich'---
+DEL ir.ui.view: crm_iap_enrich.mail_message_lead_enrich_no_credit (noupdate)
```

Comment:
This line we need to call the function `delete_records_safely_by_xml_id()` for this noupdate record to avoid error